src/components/Nav.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import hamburgerOpenMenu from '../images/hamburger-open-menu.svg'
+import brigadeLogo from '../images/cfc_logo_2021.svg'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
+
+const Nav = () => {
+  return (
+    <div>
+      <nav class="navbar" aria-label="main">
+        <button aria-label="open" onClick="toggleMenu()" class="menu-icon">
+          <img src={hamburgerOpenMenu} alt="open menu" aria-hidden="true" />
+        </button>
+        <ul class="internal-links">
```

Comment:
Make sure when using React the class property is set with `className`. This is because though it looks like we're using HTML here this is actually JSX and `class` is a reserved word in Javascript. There are a few other exceptions like this though out React.

Another interesting convention for React is that the props are all camel case instead of HTML's kabob case. Except for all the aria tags which are still supposed to be kabob case.

Reply:
Thanks for explaining this, Doug! I didn't know about the aria tags needing to be in kebab case.